src/Sleep.js

```diff
@@ -0,0 +1,106 @@
+class Sleep {
+  constructor(sleepData) {
+    this.sleepData = sleepData;
+  }
+
+  getUserData(id) {
+    return this.sleepData.filter(obj => {
+      return obj.userID === id;
+    });
+  }
+
+  calculateAverageHours(id) {
+    return this.getUserData(id).reduce((acc, object) => {
+      return Math.round(acc += object.hoursSlept / this.getUserData(id).length);
+    }, 0);
+  }
```

Comment:
We are dividing and rounding every single time, I feel like we should add them all up, then divide, then round:
```
calculateAverageHours(id) {
  let totalHours = this.getUserData(id).reduce((acc, object) => acc += object.hoursSlept, 0);
  return Math.round(totalHours / this.getUserData(id).length);
}
```

Reply:
I think we should leave the one I wrote as it gets a more accurate Math.round for our test it seems. I tried refactoring as well with .toFixed(2) to get those decimals.

Reply:
Hmmm, we can take a look at this together sometime